### ISSUE TITLE
Control over Close Button in Modal

### ISSUE THREAD

**Issue by risalfajar** [themesberg/flowbite-svelte#323]
## Summary

Allow devs to control the Close Button in a modal, for example, to hide it or disable it when needed.

### Basic example

```svelte
    <Modal title="Example" showCloseButton={false}>
        <!-- contents -->
    </Modal>
```

### Motivation

It can be used in a case where a dialog shouldn't be closed until the user complete some actions or waiting for an async operation to be finished.

**Comment by jjagielka** [comment]
Good point, but it's not about hiding the close button but about blocking the closing. Currently you can close mode by pressing Escape key or by clicking any button in the modal if the `autoclose` is set.

Anyway I'm taking that point.

**Comment by jjagielka** [comment]
@risalfajar - Can you please check the change?

**Comment by risalfajar** [comment]
By updating to v0.27.0? @jjagielka

**Comment by jjagielka** [comment]
Yes. There was a PR merged to the main trunk.

**Comment by risalfajar** [comment]
@jjagielka Just tried it. It works good! Thanks!

**Comment by Dalidos** [comment]
find the "dismissable" props on github code, need to add this in doc please :)
